### DISCUSSION TITLE
How will this project support godot 4 c# HTML5 export?

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [ashtonmeuser/godot-wasm#68] [General]
Currently godot4 c# does not support html5 export. 

Am I right?
How will this project eventually help that?

**Comment by ashtonmeuser** [comment]
Hi, @GeorgeS2019. Thanks for the question.

This project will _not_ help HTML5 exports for Godot 4 C#.

Godot Wasm focusses on running Web Assembly modules within your Godot projects rather than exporting your Godot project as a Web Assembly module (which is a requirement of HTML5 exports).

I hope this clears ups the intent of Godot Wasm.
